resolwe_bio/processes/workflows/rnaseq_variant_calling.py

```diff
@@ -0,0 +1,400 @@
+"""RNA-seq Variant Calling Workflow."""
+from resolwe.process import (
+    BooleanField,
+    Data,
+    DataField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    StringField,
+)
+from resolwe.process.models import Process as BioProcess
+
+
+class WorkflowRnaseqVariantCalling(Process):
+    """Identify variants in RNA-seq data.
+
+    This pipeline follows GATK best practices recommendantions for
+    variant calling with RNA-seq data.
+
+    The pipeline steps include read alignment (STAR), data cleanup
+    (MarkDuplicates), SplitNCigarReads, base quality recalibration
+    (BaseRecalibrator, ApplyBQSR), variant calling (HaplotypeCaller)
+    and variant filtering (VariantFiltration, SelectVariants).
+    """
+
+    slug = "workflow-rnaseq-variantcalling"
+    name = "RNA-seq Variant Calling"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {
+                "image": "public.ecr.aws/s4q6j6e8/resolwebio/dnaseq:6.3.1",
+            },
+        },
+    }
+    data_name = 'RNA-seq Variants ({{ reads|sample_name|default("?") }})'
+    version = "1.0.0"
+    process_type = "data:workflow:rnaseq:variants"
+    category = "Pipeline"
+    entity = {
+        "type": "sample",
+    }
+
+    class Input:
+        """Input fields."""
+
+        reads = DataField(
+            data_type="reads:fastq",
+            label="Input sample (FASTQ)",
+            description="Input data in FASTQ format.",
+        )
+        ref_seq = DataField(
+            data_type="seq:nucleotide", label="Reference FASTA sequence"
+        )
+        genome = DataField(
+            data_type="index:star",
+            label="Indexed reference genome",
+            description="Genome index prepared by STAR aligner indexing tool.",
+        )
+        preprocessing = BooleanField(
+            label="Perform reads processing with BBDuk",
+            default=True,
+            description="If your reads have not been processed, set this to True.",
+        )
+        dbsnp = DataField(
+            data_type="variants:vcf",
+            label="dbSNP file",
+            description="File with known variants.",
+        )
+        indels = DataField(
```

Comment:
Broad uses multiple known sites for INDELs. It would be good to support this in our pipeline as well by changing this to `ListField`. We could even join `dbsnp` with `indels` field and just extend the description with what is expected.